### PR TITLE
Responsive Site Search

### PR DESCRIPTION
**Pull request by martypowell**
Adds changes to ensure the site search lays out properly on any device size.

On smaller screens the search will be hidden and then visible when the mobile search button is selected (JavaScript required).

On larger screens, the form will always be visible in the top right.

### CODE REVIEW

**Comment by theilman7** [comment]
Currently reviewing and will provide feedback when finished.

File: src/styles/partials/components/_site-header.scss [hunk 1]
```diff
@@ -39,20 +39,34 @@
 }
 
 @media screen and (max-width: $header-breakpoint) {
```
Comment (theilman7) [inline src/styles/partials/components/_site-header.scss:43]: Recommend setting this to break at tablet size. At around 1024px (iPad size) the search bar/ `form` extends out of the frame of the header.

File: src/styles/partials/components/_site-header.scss [hunk 2]
```diff
@@ -39,20 +39,34 @@
...
+  #search-form {
+    display: none;
+    padding: 15px;
```
Comment (theilman7) [inline src/styles/partials/components/_site-header.scss]: Should use scss spacing variable`$default-padding` here for padding.

**Comment by theilman7** [comment]
@martypowell Is there a class to toggle the search form in the space the beta text appears ie. open, active etc.? Trying to preview what it looks like when the magnifying glass icon is clicked. This looks good overall.

File: src/styles/partials/components/_site-header.scss (hunk 2 above)
Comment (martypowell) [reply src/styles/partials/components/_site-header.scss]: good catch, thank you.

File: src/styles/partials/components/_site-header.scss (hunk 1 above)
Comment (martypowell) [reply src/styles/partials/components/_site-header.scss]: > @martypowell Is there a class to toggle the search form in the space the beta text appears ie. open, active etc.? Trying to preview what it looks like when the magnifying glass icon is clicked. This looks good overall.

`is-visible`

**Comment by martypowell** [comment]
> @martypowell Is there a class to toggle the search form in the space the beta text appears ie. open, active etc.? Trying to preview what it looks like when the magnifying glass icon is clicked. This looks good overall.

`is-visible`

**Review by tmccoy529 — DISMISSED** [review]
Aside from Travis comment about breakpoint size this looks good.

File: src/styles/partials/components/_site-header.scss (hunk 2 above)
Comment (martypowell) [reply src/styles/partials/components/_site-header.scss]: this has been addressed.

**Comment by martypowell** [comment]
This is pending @theilman7 sitting down with me to show the tablet issue, scheduled a meeting for tomorrow to address this.

File: src/styles/partials/components/_site-header.scss (hunk 1 above)
Comment (martypowell) [reply src/styles/partials/components/_site-header.scss:43]: @theilman7 this is happening in the components project, but looks ok when applied to actual page.

**Comment by martypowell** [comment]
@theilman7 I think this is good to go.

File: src/styles/partials/components/_site-header.scss (hunk 1 above)
Comment (theilman7) [reply src/styles/partials/components/_site-header.scss:43]: @martypowell As long as it doesn't extend out of frame when applied to the actual page, I'm fine with it. I know the components page has some weirdness when rendering some components within its containers.

**Review by theilman7 — APPROVED** [review]
Sat with Marty who showed the search bar functions fine when placed within the actual template. With this being a non-issue, looks good.

**Pull request merged**